cmake/pbft.cmake

```diff
@@ -68,6 +69,12 @@ endif()
 
 set(CMAKE_EXPORT_COMPILE_COMMANDS ON)
 
+add_e2e_test(
```

Comment:
I don't think we want to add anymore PBFT-specific tests. Ideally, this test should work for Raft too?

Reply:
Ideally it should but it won't until #589 is done. If we add it for raft now then the test will fail occasionally